tests/cluster/tests/21-many-slot-migration.tcl

```diff
@@ -40,6 +44,7 @@ test "Init migration of many slots" {
 }
 
 test "Fix cluster" {
+    wait_for_cluster_propagation
     fix_cluster $nodefrom(addr)
```

Comment:
Same here. It is shameful fix_cluster is called after wait_for_cluster_propagation! User will not "wait for cluster propagation". User will call `redis-cli --cluster fix` because his cluster is not consistent.